app/js/components/LabResultEntry.jsx

```diff
@@ -456,14 +462,22 @@ const mapStateToProps = (state) => {
     conceptMembers,
     patients,
     selectedPatient,
+    form,
   } = state;
-
+  const formId = Object.keys(form)[0];
+  let isDidNotPerformCheckboxSelected = false;
+  if (formId) {
+    const selector = formValueSelector(formId);
+    isDidNotPerformCheckboxSelected = !!(selector(state, `obs|path=did-not-perform-checkbox|conceptPath=${CONSTANTS.labResultsDidNotPerformQuestion}`));
```

Comment:
I check if the `did not perform checkbox` is checked, by checking it's value in the redux-store and coercing it to a boolean.